docs/UserGuide.md

```diff
@@ -128,37 +129,61 @@ Examples:
 Outcome Expected:
 - Success message displayed is: `New Deck added: Singapore`
 
-### Rename a new Deck : `editDeck`
+### Rename a new Deck : `editDeckName`
 
 Edit a deck name and the tags of all corresponding cards in the deck.
 
-Format: `editDeck INDEX n/NEWDECKNAME`
+Format: `editDeckName INDEX n/NEWDECKNAME`
+
+Examples:
+* `editDeckName 1 n/History` will change the name of the deck at index `1` to `History`.
+
+Outcome Expected:
+
+![EditDeckNameBefore](images/EditDeckNameBefore.png)
+
+![EditDeckNameAfter](images/EditDeckNameAfter.png)
+
+<div markdown="block" class="alert alert-warning">
 
-* Edits the deck name of the deck at the specified `INDEX`
+:warning: Things to note:
+
+* Edits the deck at the specified `INDEX`
 * The index refers to the index number shown in the currently displayed list of flashcards
 * The index **must be a positive integer** 1, 2, 3, …​
 * The `NEWDECKNAME` is case-sensitive
 
-Examples:
-* `editDeck 1 History` will change the name of the deck at index `1` to `History`.
+</div>
+
 
-Outcome Expected:
-- `to be added`
 
 ### Delete a new Deck : `deleteDeck`
 
 Deletes the deck and all the associated cards in your specified deck.
 
-Format: `deleteDeck DECKNAME`
 
-* Deletes the deck with the specified `DECKNAME`
-* Deletes all cards in the deck
+Format: `deleteDeck INDEX`
 
 Example:
-* `deleteDeck Singapore` will delete the deck and all its corresponding cards.
+* `deleteDeck 3` will delete the deck at index 3 and all its corresponding cards.
 
 Outcome Expected:
-- Success message displayed is: `Deleted deck: Singapore`
+
```

Comment:
Might be a good idea to add descriptive messages for the screen shots? e.g. 

1. Enter the deleteDeck command into the command bar as per the example
2. You should see the newly rendered list of decks will have deleted the deck away if the deck numbering given was valid.